FitNesseRoot/DbFit/AcceptanceTests/CoreTests/StoredProcNoParams/content.txt

```diff
@@ -1,5 +1,9 @@
 !3 If there are no parameters, Execute Procedure needs just one row
 
+|Clean|
+|table|clean?|
+|users|true|
```

Comment:
All the other `CoreTests` look like they create and drop the tables as part of the test page.

I'll move the `Clean` to the end of the test page.

It's an interesting point of course. When I create tests I often clean tables at the start of the test page to ensure that there's nothing hanging around from previous activities (DbFit tests or manual testing) and I guess one could consider initialising the database objects (to empty) to be set-up as well as tear-down work.